scripts/pre_push_hook.py

```diff
@@ -60,7 +60,15 @@
 FILE_DIR: Final = os.path.abspath(os.path.dirname(__file__))
 OPPIA_DIR: Final = os.path.join(FILE_DIR, os.pardir, os.pardir)
 LINTER_FILE_FLAG: Final = '--files'
-PYTHON_CMD: Final = 'python'
+
+OPPIA_IS_DOCKERIZED = bool(os.environ.get('OPPIA_IS_DOCKERIZED', False))
+if OPPIA_IS_DOCKERIZED:
+    # Path to currently running python interpreter,
+    # it is required to resolve python version conflict in docker.
+    PYTHON_CMD: Final = sys.executable
+else:
+    PYTHON_CMD: Final = 'python'
```

Comment:
Refer to https://github.com/oppia/oppia/blob/7f6c6b6d8b7767737bebeac04ade3b52d041fa53/scripts/populate_sample_contributor_data.py#L52 for a better approach.